code/modules/vending/autodrobe.dm

```diff
@@ -225,7 +225,8 @@
 		           /obj/item/clothing/under/yogs/rycliesuni = 1,
 		           /obj/item/clothing/head/yogs/toad = 1,
 		           /obj/item/clothing/head/helmet/justice = 1,
-		           /obj/item/clothing/mask/yogs/richard = 1) //yogs end
+		           /obj/item/clothing/mask/yogs/richard = 1,
+				   /obj/item/clothing/glasses/welding/jensen = 1) //yogs end
```

Comment:
Quirked up autodrobe stock?

Reply:
?

Reply:
dont these provide thermals, or was that something else

Reply:
different item